frontend/src/tests/import-export.test.js

```diff
@@ -119,6 +121,7 @@ describe("Import/export library functionality", () => {
         // ROUND TRIP TEST for prepareData function
         // create instructor CSV File object
         const instructorCSV = prepareInstructorData(instructorData, "csv");
+        // fileBits[0] stores the ArrayBuffer object of exported instructors data
```

Comment:
Why is `fileBits` an array at all? When would there be multiple items in it?

Reply:
it is because in the original code function `dataToFile` in `frontend\src\libs\import-export\data-to-file.js` makes the fileBits an array.

Reply:
according to [File API docs](https://developer.mozilla.org/en-US/docs/Web/API/File/File), it should take an array of ArrayBuffer, etc as the file content. However, since Node does not recognize `File` object and I created a shim for that, I should take `fileBits[0]` to retrieve the ArrayBuffer

Reply:
So put this explanation in the comment. Think about future readers of the code who have not done the research you have.